app/assets/javascripts/audits/batch-allocation.js

```diff
@@ -0,0 +1,38 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
+window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
+
+(function (Modules) {
+  "use strict";
+
+  Modules.BatchAllocation = function () {
```

Comment:
This module is only about allocation when it is used in that context. Maybe we could call this `SelectBatches`?